Marlin/temperature.cpp

```diff
@@ -1001,14 +1001,11 @@ void Temperature::init() {
 
   #if ENABLED(HEATER_0_USES_MAX6675)
 
-    #if DISABLED(SDSUPPORT)
-      OUT_WRITE(SCK_PIN, LOW);
-      OUT_WRITE(MOSI_PIN, HIGH);
-      SET_INPUT(MISO_PIN);
-      WRITE(MISO_PIN,1);
-    #else
-      OUT_WRITE(SS_PIN, HIGH);
-    #endif
+    OUT_WRITE(SCK_PIN, LOW);
+    OUT_WRITE(MOSI_PIN, HIGH);
+    SET_INPUT(MISO_PIN);
+    WRITE(MISO_PIN, HIGH);
+    OUT_WRITE(SS_PIN, HIGH);
```

Comment:
SS is part of the SPI hardware. If it is drawn to low from extern (somehow) the SPI hardware will go into slave mode. To prevent that the pin is made an output. To deactivate listening of a potential connected slave a high is written.